cmds/coredhcp/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	pl_router "github.com/coredhcp/coredhcp/plugins/router"
 	pl_searchdomains "github.com/coredhcp/coredhcp/plugins/searchdomains"
 	pl_serverid "github.com/coredhcp/coredhcp/plugins/serverid"
+	pl_sleep "github.com/coredhcp/coredhcp/plugins/sleep"
```

Comment:
It's probably fine since it's very small and has no dependencies or anything, but at the same time it's such a niche usecase, is there any point having it included by default ?

Maybe we should think about a checklist for choosing which plugins to build by default

Reply:
I included it because it's really small, but I agree with you that we need a clear boundary on what's admissible and what not in the "included batteries". I will merge this for now, we can change our mind later anyway